modules/search/instant-search/components/search-box.scss

```
@@ -50,4 +50,11 @@ input.jetpack-instant-search__box-input.search-field {
 .jetpack-instant-search__box input,
 .jetpack-instant-search__box input[type='search'].jetpack-instant-search__box-input {
 	width: 100%;
+
+	&::-webkit-search-decoration,
+	&::-webkit-search-cancel-button,
+	&::-webkit-search-results-button,
+	&::-webkit-search-results-decoration {
+		-webkit-appearance: searchfield;
```

```suggestion
		-webkit-appearance: searchfield;
		display: initial;
```

This should fix the issue with 2020. These elements were being set to `display: none`.

---

Brilliant @jsnmoon ! Thanks for figuring that out. Added in https://github.com/Automattic/jetpack/pull/14661/commits/9d6d6b2920cecbcc061cf28e072ac7446c41fe2b